website/zh/documents/connectors/click_house/click_house.md

```diff
@@ -0,0 +1,83 @@
+# Doris连接器
+
+上级文档: [connectors](../README.md)
```

Comment:
Great work, but there are two small things:
1. Could you provide an English version clickhouse connector document?
2. clickhouse.md and clickhouse-example.md should be saved in `website/zh/connectors/clickhouse/` directory, instead of `website/zh/documents`.

Could you also help move ftp-v1.md and ftp-v1-example.md to `website/zh/connectors/ftp/` dir?

Reply:
1. I will privide the English doc at the same time.
2. From the document path of other connectors, the correct path should be `website/zh/documents/connectors/clickhouse/`, all under the `documents/` path.
3. The ftp-v1.md and ftp-v1-example.md is wrong path, I had create an issue for it: #269

Reply:
I don't see the english docs. Did you upload them?

Reply:
Added.